trove_classifiers/__init__.py

```diff
@@ -703,6 +703,6 @@
 }
```

Comment:
First three lines of this file should be removed now?

Reply:
Good catch 👍